src/plugins/summaryWidget/src/SummaryWidget.js

```diff
@@ -87,8 +89,19 @@ define([
          * @private
          */
         function toggleTestData() {
-            self.outerWrapper.toggleClass('expanded-widget-test-data');
-            self.toggleTestDataControl.toggleClass('c-disclosure-triangle--expanded');
+            //self.outerWrapper.toggleClass('expanded-widget-test-data');

```

Comment:
Some commented code here that we should remove.

Reply:
Ah, missed this. Thank you!